soroban-env-common/src/arbitrary.rs

```diff
@@ -0,0 +1,48 @@
+//! Implementations of [`Arbitrary`] for contract types.
+
+#![cfg(feature = "testutils")]
+
+extern crate alloc;
+
+use crate::xdr::ScStatus;
+use crate::Status;
+use crate::SymbolSmall;
+use alloc::string::String;
+use alloc::vec::Vec as RustVec;
+use arbitrary::{Arbitrary, Unstructured};
+
+impl<'a> Arbitrary<'a> for SymbolSmall {
```

Comment:
What is the use case for this now? FWIW the SDK doesn't expose `SymbolSmall` explicitly and instead provides a wrapper `Symbol` type that can be either small or env-based value. So maybe it would make more sense to implement this together with SDK fuzzing support (I suppose it would be similar to fuzzing any env type, like bytes or string).

Reply:
If the SDK doesn't expose this, then it probably isn't needed. I just rebased this week and adapted this code from the prior code for Symbol. I need to go back and reevaluate which types need arbitrary impls.